build_tools/bazel/iree_lit_test.bzl

```diff
@@ -54,7 +54,7 @@ def iree_lit_test(
 
 def iree_lit_test_suite(
         name,
-        cfg = "//iree:lit.cfg.py",
+        cfg = "//iree/test:lit.cfg.py",
```

Comment:
I'm not sure there's really any reason to privilege this directory. The default made sense when everything was in a single tree using mostly the same lit file, but I don't think it really does anymore. Could make this explicit in all the lit tests under iree/test. That could be a follow-up though.

Reply:
Agreed, and I'm also wondering if we even need all these different `lit.cfg.py` files - at least a few of them are direct copies.

May address as part of the next step - moving `iree/test/` to `tests/` or some other top level directory.

Reply:
The lit tool itself requires that they be in a parent directory of the test :roll_eyes:  (unless you do some magical configuration maybe)